lib/iex/lib/iex/evaluator.ex

```diff
@@ -209,12 +209,6 @@ defmodule IEx.Evaluator do
   # line in IEx).
   #
   # Returns updated state.
-  #
-  # The first two clauses provide support for the break-trigger allowing to
-  # break out from a pending incomplete expression. See
-  # https://github.com/elixir-lang/elixir/issues/1089 for discussion.
-  @break_trigger '#iex:break\n'
-
   defp eval(code, iex_state, state) do
     try do
       do_eval(String.to_charlist(code), iex_state, state)
```

Comment:
Don't call `String.to_charlist(code)` here. Instead, pass the `code` as a string to the parser. Note you will have to change the default value of the `cache` key in the state to be `""` instead of `''`. I also recommend changing the key from `cache` to `buffer`.